components/about_build_modal/about_build_modal.jsx

```diff
@@ -10,7 +10,18 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import Constants from 'utils/constants.jsx';
 
-export default class AboutBuildModal extends React.Component {
+export default class AboutBuildModal extends React.PureComponent {
+    static defaultProps = {
+        show: false
+    };
+
+    static propTypes = {
+        show: PropTypes.bool.isRequired,
+        onModalDismissed: PropTypes.func.isRequired,
+        config: PropTypes.object.isRequired,
+        license: PropTypes.object.isRequired
```

Comment:
Would you be able to add some comments, briefly describing each prop? [Something like this](https://github.com/mattermost/mattermost-webapp/blob/master/components/post_view/post_body/post_body.jsx#L28)

Reply:
Will do.